src/library/scala/io/Source.scala

```diff
@@ -115,8 +115,9 @@ object Source {
   /** Create a `Source` from array of bytes, assuming
    *  one byte per character (ISO-8859-1 encoding.)
    */
+  @deprecated("Provide an explicit encoding", since="2.13.9")
```

Comment:
"Use `rawBytes` and provide an explicit encoding"?

Reply:
`fromBytes`, but anyone using `Source` will know that from the many hours they have wasted over the years with this API.

Reply:
`deprecated("Did you even read the Scaladoc")`

Reply:
oh, `fromBytes`, right 😬

Reply:
I force-pushed an adjusted wording.